analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -44,13 +45,12 @@ internal struct SolutionBuilder
             "System.Linq",
```

Comment:
the SolutionBuilder struct can be made readonly

Reply:
Done in #5269